components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+import type { FC } from 'react';
+import { useState, useMemo } from 'react';
+
+import Avatar from './Avatar';
+import avatarstyles from './Avatar/index.module.css';
+import styles from './index.module.css';
+
+export type AvatarType = {
```

Comment:
Why do we need to export this? And why would AvatarGroup export the type of Avatar?

If you need on Storybook you can do it with:

```tsx
import type { ComponentProps } from 'react';

type AvatarProps = ComponentProps<typeof Avatar>;
```